man/org.freedesktop.systemd1.xml

```diff
@@ -1434,7 +1434,12 @@ node /org/freedesktop/systemd1 {
 
       <para><function>Subscribe()</function> enables most bus signals to be sent out. Clients which are
       interested in signals need to call this method. Signals are only sent out if at least one client
-      invoked this method. <function>Unsubscribe()</function> reverts the signal subscription that
+      invoked this method. Note that this behavior is applied to all the signals that the object
+      in <constant>/org/freedesktop/systemd1</constant> may emit, including the
+      DBus standard <constant>org.freedesktop.DBus.Properties.PropertiesChanged</constant>
+      signal that technically do not belong to this interface.</para>
```

Comment:
`signals` as this refers to multiple ones. Also please leave the consideration about whether or not it belongs here out, as it's up for debate and not really important to have as part of this manpage

Reply:
I had used `signals` before, but that's just a signal name... I've reworded the second part, but it's important imho to make clear that this also controls the signals that are *by definition* part of the `org.freedesktop.systemd1` dbus interface, but of other ones.